src/components/auth/login/loginContainer.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { toast } from 'react-toastify';
+import Login from './Login';
+import 'react-toastify/dist/ReactToastify.css';
+import loginUserAction from '../../../actions/auth/loginAction';
+
+export class LoginContainer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: '',
+      password: '',
+      isLoading: false,
+    };
+  }
+
+  onInputChange = (e) => {
+    e.preventDefault();
+    this.setState({
+      [e.target.name]: e.target.value,
+    });
+  }
+
+  handleSubmit = async (e) => {
+    this.setState({ isLoading: true });
+    e.preventDefault();
+    const { email, password } = this.state;
+    const loginInfo = {
+      email,
+      password,
+    };
+    const { loginAction } = this.props;
+
+    if (this.validateEmailPasswordEntry(email, password)) {
+      return;
+    }
+
+    await loginAction(loginInfo);
+
+    this.setState({ isLoading: false });
+
+    this.handleClearForm();
+  }
+
+  validateEmailPasswordEntry(email, password) {
+    if ((email === '') || (password === '')) {
+      toast.error('Both Email & Password Details Required', {
+        containerId: 'A',
+        position: toast.POSITION.TOP_RIGHT,
+        autoClose: false,
+        hideProgressBar: false,
```

Comment:
@Manorlds-Eaglespark, Please consider using the toast within the util.js file

Reply:
Sure, thanks.